### ISSUE TITLE
pipeline failing with An error occurred while provisioning resources (Error Type: Failure). The remote provider was unable to process the request.  

### ISSUE THREAD

**Issue by somiya9868** [actions/virtual-environments#2575]
**Description**  
The azure pipeline is failing with error 
```
"An error occurred while provisioning resources (Error Type: Failure). The remote provider was unable to process the request."  
```
**Area for Triage**:  

**Question, Bug, or Feature?**:  
Bug

**Virtual environments affected**  
- [ ] Image: ubuntu-latest

**Expected behavior**  
The pipeline should have executed with no issues.

**Actual behavior**  
Suddenly few pipelines stopped running with the mentioned error.

**Repro steps**  
A description with steps to reproduce the issue. If your have a public example or repo to share,
please provide the link.

1. ran the pipeline with Image: ubuntu-latest

**Comment by andy-mishechkin** [comment]
Hello, @somiya9868 
May you provide link to failed pipeline, please

**Comment by miketimofeev** [comment]
@somiya9868 there was an issue with the backend service, which is resolved by now. Sorry for the inconvenience.
I'm going to close the issue, but please feel free to contact us if you have any concerns.
Thank you!

**Comment by mottykohn** [comment]
I'm having same issue right now. I checked https://status.azure.com/en-us/status/ and there is no indication of anything being down

**Comment by MattJeanes** [comment]
Yep same issue here, I just made a big config change so I thought I broke something 😁

**Comment by sentimental37** [comment]
Yes, for me as well, just finished converting EF 6 to EF Core, and tried building the branch, but got a build canceled email. Nothing updated on Azure status page.

**Comment by Aaronmyster** [comment]
Also having the same issue right now.

**Comment by jessebacon** [comment]
Same issue
```
##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
```

**Comment by calvaradocl** [comment]
Same issue from my side. The status of the pipelines are being reported as Healthy.

##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: ubuntu-18.04
Started: Just now
Duration: 15s

Job preparation parameters
1 queue time variable used

**Comment by alelfreitas** [comment]
Same here...

**Comment by waylonmtz** [comment]
we are having this issue too

**Comment by andrewwiebe** [comment]
Yup same issue here

**Comment by GuerreiroLeonardo** [comment]
Same here

**Comment by kummerer94** [comment]
Same issue.

**Comment by wesselkranenborg** [comment]
Same here

**Comment by aphavichitr** [comment]
Same here

**Comment by jwyglendowski-precisionlender** [comment]
Same here

**Comment by mvanhulzen** [comment]
Same here, My fix was reverting to ubuntu-16.04. that did work

**Comment by ldelpozo** [comment]
same here

**Comment by TejshreeYeole** [comment]
same here

**Comment by wesselkranenborg** [comment]
> Same here, My fix was reverting to ubuntu-16.04. that did work

That's not a fix @mvanhulzen  ;-)

**Comment by MartynasKer** [comment]
same

**Comment by warleipereira** [comment]
same here

**Comment by MattJeanes** [comment]
What Azure DevOps region is everyone in? React to this comment:

👍🏻 - United States
👎🏻 - Canada
😄 - Brazil
🎉- Europe
😕 - United Kingdom
❤️ - Asia Pacific
🚀 - Australia
👀 - India

**Comment by MattJeanes** [comment]
What OS agent region is everyone in? React to this comment:

👍🏻 - United States
👎🏻 - Canada
😄 - Brazil
🎉- Europe
😕 - United Kingdom
❤️ - Asia Pacific
🚀 - Australia
👀 - India

**Comment by karsegri** [comment]
Same issue over here!

**Comment by acheraime** [comment]
Same issue here.

**Comment by maximhazankin** [comment]
Same here

**Comment by nigreany** [comment]
Same here - United States

##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: windows-latest
Started: Today at 2:04 PM
Duration: 5m 2s

Job preparation parameters

**Comment by yellowcitytech** [comment]
Someone broke something for sure....

**Comment by MattJeanes** [comment]
What agent is everyone using? React to this comment:

Just trying to collate some info to see the severity of this outage in the absence of the status page saying anything. I'm not Microsoft or anything!

👍🏻 - macOS-10.14
👎🏻 - macOS-10.15
😄 - ubuntu-16.04
🎉- ubuntu-18.04
😕 - ubuntu-20.04
❤️ - vs2017-win2016
🚀 - windows-2019

**Comment by taha-azzabi** [comment]
Same here Europe, and what a chance I had, were planing a huge release on prod tonight

**Comment by shaunlavallee** [comment]
Same problem here .. :(

**Comment by ttaylor29** [comment]
Same here in US - Alabama.

**Comment by ramdil** [comment]
same problem, we are in US vs2017-win2016

**Comment by asehgalxom** [comment]
Same problem here, thought I broke something until I saw this thread lol

**Comment by yuriduque-ciandt** [comment]
mesmo problema aqui

**Comment by maxxychen** [comment]
same problem, vs2017-win2016, East US

**Comment by choksz** [comment]
same issue I thought I broke something on my end. tried ubuntu-latest and windows-latest no luck.

**Comment by jeev20** [comment]
Yup down here (Norway) as well. Build fails.

**Comment by digimaun** [comment]
Concur, having the same issue here.

**Comment by gavinsteinhoff** [comment]
Looks like are all having errors today.

##[error]An error occurred while provisioning resources (Error Type: Failure).
##[error]The remote provider was unable to process the request.

**Comment by MarkMastronardi** [comment]
Same here.

**Comment by gorziza** [comment]
East US
![Selection_006](https://user-images.githubusercontent.com/8699254/126547839-05c5621e-bfbf-4387-b78a-a375fb6a0a45.png)

**Comment by elpablete** [comment]
Same here!!!

**Comment by cristian-conte** [comment]
Same here :(

**Comment by LR4Discovery** [comment]
Same issue in North Central

**Comment by LucasFievet** [comment]
Same here :(

**Comment by acons-TCA** [comment]
Same on East US

**Comment by chiller-whale** [comment]
Same here :(

**Comment by NicoleSelig** [comment]
Same here -- East US

**Comment by saiiii** [comment]
Same all over the world!

**Comment by jjcaraballo01** [comment]
Same issues here Central US. Well so much for that deadline we're trying to hit converting all Bamboo and Octopus Deploy plans to YAML pipelines in ADO by eow ;(

**Comment by lashajala** [comment]
Same West Europe

**Comment by anderson-fachini** [comment]
Same in Brazil with Ubuntu 16

**Comment by paul-t-wright** [comment]
> What agent is everyone using? React to this comment:
> 
> Just trying to collate some info to see the severity of this outage in the absence of the status page saying anything. I'm not Microsoft or anything!
> 
> 👍🏻 - macOS-10.14
> 👎🏻 - macOS-10.15
> 😄 - ubuntu-16.04
> 🎉- ubuntu-18.04
> 😕 - ubuntu-20.04
> ❤️ - vs2017-win2016
> 🚀 - windows-2019

ubuntu-latest

**Comment by nathanjhill1** [comment]
We are also having the same issue. We are in South Central US. We are using macOs-10.14

**Comment by sanjay3290** [comment]
same issue in central us

**Comment by warleyxavier** [comment]
Same here 😕

**Comment by KaiserZoza** [comment]
same here :(

**Comment by asehgalxom** [comment]
This is has been closed so maybe no one is paying attention, I opened a new issue. Everyone please try to raise that up

**Comment by fbuzila** [comment]
the same here:

##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: Ubuntu16

**Comment by MattJeanes** [comment]
Cheers for voting above everyone, according to the votes it looks like this is a complete worldwide outage across all agents 😨

This is a closed issue but it should still be sending notifications to maintainers, I recommend raising critical severity tickets on Azure support if this is severely impacting you to get some attention to this issue, nothing official yet. I've DM'd @AzureDevOps on Twitter for what it's worth but I am not severely impacted as it's out of work hours for us and we don't critically need them at the moment.

**Comment by jason-e-gross** [comment]
samesies

**Comment by christianAtSuddath** [comment]
yes:
```
##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: windows-2019
Started: Just now
Duration: <1s
```

**Comment by davidnarciso** [comment]
Same here, Dev team blocked from builds :(

**Comment by VikingHobo** [comment]
Having the issue as well

**Comment by prateekmanochakwc** [comment]
same here region north europe

**Comment by IgooorGP** [comment]
Same thing here. Region: Brazil South.

**Comment by sreepriyaks** [comment]
same in Australia East right since more than half an hour

**Comment by goleafs** [comment]
yep, same for us, East Canada
no problems showing here - https://status.dev.azure.com/

**Comment by sawaYch** [comment]
Same. 
Region: East Asia
Image: ubuntu-latest

**Comment by sergio3454** [comment]
Same problem here. Region: Brazil South.

**Comment by christianAtSuddath** [comment]
> yep, same for us, East Canada
> no problems showing here - https://status.dev.azure.com/

Unbelievable!!!

**Comment by evanwhitten** [comment]
Same problem here "The remote provider was unable to process the request."

**Comment by jsalmeida1101** [comment]
Same here

**Comment by AlexandreMuskus** [comment]
https://developercommunity.visualstudio.com/t/errorAn-error-occurred-while-provisi/1481083?space=21&ftype=problem&stateGroup=active&sort=newest

**Comment by tessmanjonathan** [comment]
Seeing the same problem across multiple different orgs

**Comment by gmurched** [comment]
same here, Brazil

**Comment by SorenKlemmensen** [comment]
Same issue. Region: Canada Central, vmImage: Windows-Latest

**Comment by ssaric** [comment]
Same here -> West Europe

**Comment by praath** [comment]
We are also experiencing this issue in a Build pipeline.

East US 2
Visual Studio version: latest

**Comment by billybraga** [comment]
> https://developercommunity.visualstudio.com/t/errorAn-error-occurred-while-provisi/1481083?space=21&ftype=problem&stateGroup=active&sort=newest

There are other issues that were just created also:

https://developercommunity.visualstudio.com/t/Pipelines-unable-to-get-agent-from-MS-ho/1481067?space=21&q=%22The+remote+provider+was+unable+to+process+the+request%22

https://developercommunity.visualstudio.com/t/Pipeline-Build-on-Hosted-agent-fails-wit/1481043?space=21&q=%22The+remote+provider+was+unable+to+process+the+request%22

The problem is that there is not the place to report service availability problems:

![image](https://user-images.githubusercontent.com/426739/126549524-80f5421b-56db-4fb0-975c-69f953c541e9.png)

https://docs.microsoft.com/en-us/azure/devops/user-guide/service-status-info?view=azure-devops#:~:text=report%20your%20issue%20using%20the%20Report%20a%20Service%20Availability%20Issue%20form

But there is no link to the **Service Availability form**

**Comment by MateusMo** [comment]
Same here, Brazil

**Comment by davidbirt** [comment]
Same issue.

**Comment by jdunnink** [comment]
same issue here

**Comment by kcherewyk** [comment]
Same issue in western Canada

**Comment by hosseinis** [comment]
Same issue here from U.S.

**Comment by ssay-work** [comment]
same issue using a hosted agent in eastern us

**Comment by jwyglendowski-precisionlender** [comment]
![image](https://user-images.githubusercontent.com/40304875/126550373-b2152d23-13f1-467b-b249-0bceade46bae.png)
Looks like they are finally looking into it.

**Comment by davinomjr** [comment]
Same issue

**Comment by betabrandao** [comment]
same issue in Brazil.

**Comment by Qi-Jian-Huang-DevOps** [comment]
Oh nice, I was trying to figure out what I did wrong for the past one hour, thought I broke something,  ended up is something wrong from the hosted agent. Interesting.

Agent: windows-latest
region: canada central

**Comment by wggley** [comment]
My fellow developers, maybe you are all trying to figure why I have called you here...

+1 Brazil

**Comment by cricketthomas** [comment]
same issue.

**Comment by hilary-b** [comment]
Same issue in Eastern US

**Comment by alasdairmackenzie** [comment]
They've updated the status page now https://status.dev.azure.com

**Comment by christianAtSuddath** [comment]
![image](https://user-images.githubusercontent.com/19861139/126551083-3b4f1439-8cc2-4b49-9926-50323f604cf4.png)

**Comment by bcbeatty** [comment]
https://status.dev.azure.com/_event/251790226

**Comment by maikelvanhaaren** [comment]
Fellow developer, what's your favorite programming joke? 

Mine: Why Java developers have glasses? Because they can't see sharp. Ba dum tss..

**Comment by betabrandao** [comment]
![image](https://user-images.githubusercontent.com/55194640/126554955-7b641b2a-b844-43e1-a01f-bb0a29c7d5f5.png)


> Fellow developer, what's your favorite programming joke?
> 
> Mine: Why Java developers have glasses? Because they can't see sharp. Ba dum tss..

**Comment by Skirtek** [comment]
> Fellow developer, what's your favorite programming joke?
> 
> Mine: Why Java developers have glasses? Because they can't see sharp. Ba dum tss..

SQL database walked into a NoSQL bar. A little while later he walked out, because he could not find a table :)

**Comment by praath** [comment]
> We are also experiencing this issue in a Build pipeline.
> 
> East US 2
> Visual Studio version: latest

This is resolved on my end.

**Comment by sentimental37** [comment]
It started working for me 
![image](https://user-images.githubusercontent.com/6756807/126555617-c092244b-369c-4906-a560-c8b10021f7ca.png)

**Comment by suboorjamalredmane** [comment]
It is resolved in Central US now.

**Comment by MateusMo** [comment]
Thank you Bill Gates, It's working here in Brazil

**Comment by wggley** [comment]
Can someone please point me out where we can create tickets for M$?

As everyone here figured we couldn't find a place to report and googled here.

+1 Brazil

**Comment by ajsharmaofficial** [comment]
Seems this is not fully fixed yet. Started seeing it again.

**Comment by Qi-Jian-Huang-DevOps** [comment]
@ajsharmaofficial canada central is working fine.

**Comment by ajsharmaofficial** [comment]
![image](https://user-images.githubusercontent.com/33105726/126909733-c32edcb6-2f55-4b1e-a0e0-63a55fe00d32.png)


Getting this error on canada central. is there something else could cause the same error. I could be missing something.

**Comment by ajsharmaofficial** [comment]
Just changed to a self hosted agent and its working fine. There might be something wrong on Canada central Microsoft hosted agent.

**Comment by Qi-Jian-Huang-DevOps** [comment]
I am in canada central, and it is working fine for me, I just tested it.
I am using windows-latest hosted agent.

**Comment by andrewwiebe** [comment]
This was corrected last week, however it just started happening again this morning. Location CANADA, build agent linux latest.

**Comment by andrewwiebe** [comment]
I just restarted the failed build and its fine now. Maybe intermittent?

**Comment by pleymor** [comment]
Looks like it's back! I've got this error for an hour (west europe).

Europe is supposed to be healthy yet according to https://status.dev.azure.com/.

Logs:

```
##[error]This is a scheduled Ubuntu 16.04 brownout. Ubuntu 16.04 LTS environment will be removed on September 20, 2021. For more details, see https://github.com/actions/virtual-environments/issues/3287
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: ubuntu-16.04
Queued: Just now [manage parallel jobs]
Started: Just now
Duration: 59s

Waiting on the agent to receive and acknowledge the request...
Job preparation parameters
ContinueOnError: False
TimeoutInMinutes: 60
CancelTimeoutInMinutes: 5
Expand:
  MaxConcurrency: 0
  ########## System Pipeline Decorator(s) ##########

  Begin evaluating template 'system-pre-steps.yml'
Evaluating: eq('true', variables['system.debugContext'])
Expanded: eq('true', Null)
Result: False
Evaluating: resources['repositories']['self']
Expanded: Null
Result: False
Finished evaluating template 'system-pre-steps.yml'
********************************************************************************
Template and static variable resolution complete. Final runtime YAML document:
steps: []
```

**Comment by maxim-lobanov** [comment]
@pleymor , pay attention on error message: 
##[error]This is a scheduled Ubuntu 16.04 brownout. Ubuntu 16.04 LTS environment will be removed on September 20, 2021. For more details, see https://github.com/actions/virtual-environments/issues/3287

**Comment by pleymor** [comment]
Ok, so I guess it's the same, but impacting only the teams who haven't migrated their agent yet (probably an opération before the end of the support on septembre 20).

**Comment by alexz76** [comment]
It appears this issue is back! I'm getting the error below when running my pipeline using agent pool "Azure Pipelines". No changes happened in my `azure-pipelines.yml` file and just running a simple echo job fails as well.

**Error message:**

```
##[warning]An image label with the label Ubuntu16 does not exist.
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: Ubuntu16

Job preparation parameters
Variables:
  FrontendAuthenticationUri: $[ variablegroups.tmo-k8s-variables.FrontendAuthenticationUri ]
  RepositoryUri: $[ variablegroups.tmo-k8s-variables.RepositoryUri ]
  RepositoryUriList: $[ variablegroups.tmo-k8s-variables.RepositoryUriList ]
  TAG: $[ variablegroups.tmo-k8s-variables.TAG ]
  TemplatesUrl: $[ variablegroups.tmo-k8s-variables.TemplatesUrl ]
  VERSION: $[ variablegroups.tmo-k8s-variables.VERSION ]
ContinueOnError: False
TimeoutInMinutes: 60
CancelTimeoutInMinutes: 5
Expand:
  MaxConcurrency: 0
  ########## System Pipeline Decorator(s) ##########

  Begin evaluating template 'system-pre-steps.yml'
Evaluating: eq('true', variables['system.debugContext'])
Expanded: eq('true', Null)
Result: False
Evaluating: resources['repositories']['self']
Expanded: Object
Result: True
Evaluating: not(containsValue(job['steps']['*']['task']['id'], '6d15af64-176c-496d-b583-fd2ae21d4df4'))
Expanded: not(containsValue(Object, '6d15af64-176c-496d-b583-fd2ae21d4df4'))
Result: True
Evaluating: resources['repositories']['self']['checkoutOptions']
Result: Object
Finished evaluating template 'system-pre-steps.yml'
********************************************************************************
Template and static variable resolution complete. Final runtime YAML document:
steps:
- task: 6d15af64-176c-496d-b583-fd2ae21d4df4@1
  inputs:
    repository: self


  MaxConcurrency: 0
```

**YAML Sample:**

```yaml
name: "Backend API $(Build.BuildId)"

variables:
  - group: tmo-k8s-variables

trigger:
  branches:
    include:
      - release/*
      - hotfix/*
      - feature/*
      - main

stages:
  - stage: testing_stage
    displayName: "🧪 Testing"
    jobs:
      - job: testit
        displayName: "Testing"
        steps:
          - bash: |
              echo "Unit tests should run here."
            displayName: "Testing"
```

Does anyone have an idea? Both Azure status page related to Azure DevOps shows it as healthy.

- **https://status.dev.azure.com/**

![image](https://user-images.githubusercontent.com/907751/137988603-575b58d0-2dbc-4654-9f74-34d9d80d3a0b.png)

- **https://status.azure.com/en-us/status/**

![image](https://user-images.githubusercontent.com/907751/137988674-a7f4fbeb-99e4-447c-87b1-50a892aca064.png)

**Comment by leonardBang** [comment]
Thanks for your @miketimofeev response, I've fixed this issue,  and the docs[1] should update too
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

**Comment by gene-keys** [comment]
@miketimofeev there is no working solution on that post, already using "vmImage: 'ubuntu-latest'", changing to Azure Pipelines in my region results in 503 service not available. What is likely happening is the issue is regional and intermittent.

**Comment by Sushma-tiwari** [comment]
Mine is **Central US** check to snap :
[https://prnt.sc/1y1xjvm](url)

But But still getting this error from past 7 days : 
##[warning]An image label with the label ubuntu-16.04 does not exist.
,##[error]The remote provider was unable to process the request.

Check error snap : 
[https://prnt.sc/1y1xped](url)

Thanks,

**Comment by crhistianramirez** [comment]
The ubuntu retirement may be the cause for some users but I am running windows 2019 vm and still seeing it crop up.

**Comment by NeerajaGandla** [comment]
Same issue here. I don't use Ubuntu. I use Windows 10. Did anyone find a resolution?

**Comment by crhistianramirez** [comment]
@NeerajaGandla for me it was because I was using azure-pipelines.yml file and while I had defined pool.vmImage for one of my jobs, I had left it blank for two others and for some reason it was pulling in the old failed ubuntu version. You need to explicitly set pool.vmImage if you are not currently

[Link to the commit that fixed my issue](https://github.com/ordercloud-api/headstart/commit/0b288a60acd850352d1ab70f926379943d81e965)

**Comment by BrainSlugs83** [comment]
@miketimofeev -- it seems that his issue has regressed today in Central US region.  

There is no outage reported on [https://status.azure.com/en-us/status/](https://status.azure.com/en-us/status/) that I can see. -- 1. Can you please update the Azure outage page, or re-open this issue?
2. The error I recieve is the exact same error message and it is very vague. -- It seems the real issue is that the error is not actionable and it will always regress back to this bug because it's the same error message as this one. -- Can you please update the error messages to be less vague so that we can do something actionable?  (or where we can open a new bug without it being a duplicate of this one?)

```
##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: [Azure Pipelines](https://dev.azure.com/REDACTED/REDACTED/_settings/agentqueues?poolId=&queueId=27)
Image: windows-latest
Started: Today at 10:11 AM
Duration: 6m 9s
```

**Comment by schafric** [comment]
@BrainSlugs83 , @miketimofeev we have the same issue. Status page acknowledges nothing as you wrote @BrainSlugs83 🤷🏻‍♂️

**Comment by coolhome** [comment]
I can confirm it's been having pretty sporadically in the last few hours for Microsoft Hosted Agents. I wish the pipelines would re-try when this type of error occurs.

**Comment by softwareliberationarmy** [comment]
Same here. Seeing this issue in the Central US region since yesterday.

**Comment by stevebumbaugh** [comment]
Seeing this in Central US region as well, intermittently today

**Comment by paillave** [comment]
Since today (the 13th february), in Europe, I face this issue in 75% of my builds. It seems to happen with heavy CPU build (TypeScript for example)

**Comment by ahilmathew** [comment]
Same issue happening now

**Comment by ksooner** [comment]
getting same error in Australia region
`##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.`

**Comment by tknaller** [comment]
same here west europe

**Comment by Lkraehen** [comment]
+1 Australia region

##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.

**Comment by landre14** [comment]
Same here in the US using 'windows-latest' hosted agent

**Comment by michaelcohenunsw** [comment]
+1 Australia region

##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.

**Comment by shaandre96** [comment]
+1 Australia region

##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.

**Comment by kchilka-msft** [comment]
+1 same here, in US region 🥲

**Comment by nanonandi** [comment]
Same issue weu :(

**Comment by ko16a46** [comment]
Is there an official way to report this pipeline host downtime issue to Azure Devops? 
We're getting the same issue here in East US 2 region but there's no reported downtime in https://status.dev.azure.com/

```
##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[error]Provisioning request delayed or failed to send 5 time(s). This is over the limit of 3 time(s).
```

**Comment by tknaller** [comment]
my google fu tells me we should be telling this to https://azure.microsoft.com/en-us/support/devops/ - but there's only an error aswell.

**Comment by jmeraq** [comment]
Mismo problema por aqui !! Heeeeeeeeeeeeeeelp !!

**Comment by landre14** [comment]
I created an issue with Microsoft here: https://developercommunity.visualstudio.com/t/Azure-Pipeline-hosted-agents-are-down-w/10298058?q=azure+pipeline+hosted+agent

**Comment by germanmartin** [comment]
We are expecting the same issues! I opened an internal ticket to microsoft with A severity

**Comment by Gustavo-Marques19** [comment]
same here

`##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[warning]There was a failure in sending the provision message: An unexpected failure occurred while attempting to contact the remote provider.
,##[error]Provisioning request delayed or failed to send 5 time(s). This is over the limit of 3 time(s).
Pool: [Azure Pipelines](https://dev.azure.com/DevOpsVBB/d13ea5be-8198-4358-9a51-c13c09294942/_settings/agentqueues?poolId=&queueId=38)
Image: windows-2022
Started: Today at 20:37
Duration: 45m 37s`

**Comment by hanguyentgg** [comment]
+1 Australia

**Comment by GFoley83** [comment]
Just goes to show, as ever, the status page is absolutely USELESS only only updated after everyone knows there's an issue. Infuriating.

**Comment by hamenu** [comment]
Apparently, no one looks at a closed issue 😒 

I've opened a new one, let's see if anyone looks at that one _eventually_:

https://github.com/actions/runner-images/issues/7228

**Comment by kchilka-msft** [comment]
I tried running it again (US region) and it worked for me now! I didnt make any changes btw

**Comment by BrainSlugs83** [comment]
Same.  Started working again without any changes on my end. -- It was broken for a few days and then started working again.  Never noticed any reported outage either. :(
